tools/scan-build-py/tests/unit/test_analyze.py

```diff
@@ -353,63 +353,63 @@ def test_no_map_gives_empty(self):
         self.assertFalse(pairs)
 
     def test_multiple_maps_merged(self):
-        concat_map = ['_Z1fun1i@x86_64 ast/x86_64/fun1.c.ast',
-                      '_Z1fun2i@x86_64 ast/x86_64/fun2.c.ast',
-                      '_Z1fun3i@x86_64 ast/x86_64/fun3.c.ast']
+        concat_map = ['_Z1fun1i ast/fun1.c.ast',
```

Comment:
Maybe tests should no longer use mangled names but USRs.

Reply:
I'll change it.